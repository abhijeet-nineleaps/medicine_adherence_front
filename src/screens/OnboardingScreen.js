import React from 'react';
import {View, Image} from 'react-native';
import styles from './screenStyles/OnboardingStyles';

const OnboardingScreen = (navigation) => {
  setTimeout(() =>  {
    navigation.navigate('Drawer');
  }, 500);
  return (
    <View
      style={styles.container}>
      <Image
        source={require('../../assests/images/Medstick_1.png')}
        style={styles.img}
        resizeMode="contain"></Image>
    </View>
  );
};

export default OnboardingScreen;
