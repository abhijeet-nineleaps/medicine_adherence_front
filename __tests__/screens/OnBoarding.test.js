import React from 'react';
import renderer from 'react-test-renderer';
import OnboardingScreen from '../../src/screens/OnboardingScreen';
jest.mock("@react-native-google-signin/google-signin", () => ({
  default: jest.fn(),
}));
jest.mock("@react-navigation/native", () => ({
  ...jest.requireActual("@react-navigation/native"),
  useFocusEffect: jest.fn(),
}));
describe('OnBoarding Screen', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<OnboardingScreen/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
