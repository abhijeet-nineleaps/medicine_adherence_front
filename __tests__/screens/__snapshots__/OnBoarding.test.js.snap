// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnBoarding Screen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#3743ab",
      "height": "100%",
      "justifyContent": "center",
    }
  }
>
  <Image
    resizeMode="contain"
    source={
      Object {
        "testUri": "../../../assests/images/Medstick_1.png",
      }
    }
    style={
      Object {
        "height": 400,
        "marginTop": 20,
        "width": 400,
      }
    }
  />
</View>
`;
